src/structures/Guild.js

```diff
@@ -770,6 +793,22 @@ class Guild extends Base {
     );
   }
 
+  /**
+   * Edits the guild's embed.
+   * @param {GuildEmbedData} embed The embed for the guild
+   * @param {string} [reason] Reason for changing the guild's embed
+   * @returns {Promise<Guild>}
```

Comment:
Discord respond here with `{ channel_id: Snowflake | null, enabled: boolean }`, not the Guild.
Maybe explicitly resolve with the Guild or map the data to a proper GuildEmbedData instead?